pkg/controllers/provisioning/scheduling/nodeclaimtemplate.go

```diff
@@ -77,5 +86,22 @@ func (i *NodeClaimTemplate) ToNodeClaim(nodePool *v1beta1.NodePool) *v1beta1.Nod
 		Spec: i.Spec,
 	}
 	nc.Spec.Requirements = i.Requirements.NodeSelectorRequirements()
-	return nc
+	return nc, nil
+}
+
+func FindIncompatibleRequirementKeyAcrossInstanceTypeOptions(requirements scheduling.Requirements, instanceTypes cloudprovider.InstanceTypes) string {
```

Comment:
Is there a way to shorten this function name? I get that the function name is highly descriptive, but it feels like an immediate code smell when we start getting function names this long.